### PR TITLE
Assorted cleanup

### PR DESCRIPTION
**Pull request by apasel422**

### CODE REVIEW

**Comment by FlashCat** [comment]
r? @Gankro

(I've picked a reviewer for you, use r? to override)

**Comment by Gankra** [comment]
@FlashCat r+

**Comment by FlashCat** [comment]
:pushpin: Commit 36c0c74 has been approved by `Gankro`

**Comment by FlashCat** [comment]
:hourglass: Testing commit 36c0c74 with merge e904151...

**Comment by apasel422** [comment]
Did this get stuck?

**Comment by Gankra** [comment]
Oh. Huh. CC @huonw and @barosl if you want to collect debug info.

**Comment by Gankra** [comment]
Honestly using r+ for these repos is reaaaaally gratuitous and silly; it was just kinda fun to test out.

**Comment by Gankra** [comment]
Hmm, I'm not sold on `peek_min`. What's the argument against plain `min`?

**Comment by Gankra** [comment]
Relevant to Ordered Map APIs, I reckon.

**Comment by apasel422** [comment]
Hmm, no argument there. I was just going for consistency with `BinaryHeap`, which has `peek` instead of `max`.

**Comment by apasel422** [comment]
And you're right, it's definitely relevant to ordered maps and sets, although I've been expecting ordered maps to provide `remove_min`. Should they call it `pop_min` instead?

**Comment by Gankra** [comment]
Hmm @sellibitze any opinions?

**Comment by Gankra** [comment]
@reem and @aturon seem in favour of just `min`. Added bonus of symmetry between queues and maps (which are actually a stronger priority queue anyway), for what that's worth.

**Comment by apasel422** [comment]
Done.

File: src/lib.rs [hunk 1]
```diff
@@ -205,14 +209,14 @@ impl<T: Ord> IntervalHeap<T> {
...
-    /// ```rust
+    /// ```
     /// # extern crate "interval-heap" as interval_heap;
     /// # fn main() {
     /// use interval_heap::IntervalHeap;
     ///
     /// let heap = IntervalHeap::from_vec(vec![5, 1, 6, 4]);
     /// assert_eq!(heap.len(), 4);
-    /// assert_eq!(heap.get_min_max(), Some((&1, &6)));
+    /// assert_eq!(heap.min_max(), Some((&1, &6)));
```
Comment (Gankra) [inline src/lib.rs:219]: One more letter and we can get some sweet tree search algorithms in this repo :dancer:

**Pull request merged**